src/test/groovy/com/google/gradle/osdetector/OsDetectorPluginTest.groovy

```diff
@@ -31,5 +32,21 @@ class OsDetectorPluginTest {
     assertNotNull(project.osdetector.arch)
     assertEquals(project.osdetector.os + '-' + project.osdetector.arch,
         project.osdetector.classifier)
+    System.err.println('classifier=' + project.osdetector.classifier)
```

Comment:
Because the test flow depends on the system it is run, I intend to keep these printouts to double check that the flow goes in the right way.